src/test/java/seedu/us/among/testutil/EditEndpointDescriptorBuilder.java

```diff
@@ -43,9 +43,9 @@ public EditEndpointDescriptorBuilder(Endpoint endpoint) {
 
     /**
      * Sets the {@code Name} of the {@code EditEndpointDescriptor} that we are
-     * building. //to-do refactor this from withName to withMethod (CommandTestUtil Class)
+     * building.
      */
-    public EditEndpointDescriptorBuilder withName(String name) {
+    public EditEndpointDescriptorBuilder withMethod(String name) {
```

Comment:
Since the method name has changed perhaps the parameter name should be updated as well

Reply:
done brudda